src/System.Private.CoreLib/shared/System/Threading/SynchronizationContext.cs

```diff
@@ -0,0 +1,64 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Thread = Internal.Runtime.Augments.RuntimeThread;
+
+namespace System.Threading
+{
+    public partial class SynchronizationContext
+    {
+        private bool _requireWaitNotification;
+
+        public SynchronizationContext()
+        {
+        }
+
+#if !FEATURE_APPX
```

Comment:
This should be `#if !FEATURE_APPX && !ENABLE_WINRT` to make it work for ProjectN.